server/vulnerabilities/msrc/sync.go

```diff
@@ -58,10 +58,12 @@ func bulletinsDelta(
 	return toDownload, toDelete
 }
 
-// Sync syncs the local msrc security bulletins (contained in dstDir) for one or more operating systems with the security
-// bulletin published in Github.
+// SyncFromGithub syncs the local msrc security bulletins (contained in dstDir) for one or more operating
+// systems with the security bulletin published in Github.
+//
 // If 'os' is nil, then all security bulletins will be synched.
-func Sync(ctx context.Context, client *http.Client, dstDir string, os []fleet.OperatingSystem) error {
+func SyncFromGithub(ctx context.Context, dstDir string, os []fleet.OperatingSystem) error {
+	client := fleethttp.NewGithubClient()
```

Comment:
Because in all occurrences of these methods we were providing `fleethttp.NewClient()`, so why expose as argument?

Reply:
Usually, you would want to avoid initializing a new http client on every request, but since this function is called infrequently it wouldn't make much difference.

> Should we update `download.DownloadAndExtract` to do the same?

Disregard, obviously this should take a client ...